src/App/ComponentsDocumentation/components/SkipLink/index.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import { ComponentPreview, DocContainer } from "@docutils";
+import { skipLinkShowcase } from "./constants";
+import SkipLinkComponent from "@components/SkipLink";
+import CodeTags from "@components/CodeTags";
+
+const Overview = () => (
+    <section id="skip-link-overview">
+        <h2 id="overview">Overview</h2>
+        <ComponentPreview language="html" showCasePanel showCasePanelAdvanced={skipLinkShowcase} hideCodeFigure/>
+        <ComponentPreview language="html" codeFigure><SkipLinkComponent/></ComponentPreview>
+    </section>
+);
+
+const DeveloperDocumentation = () => (
+    <section>
+        <h2 id="developer-documentation">Developer Documentation</h2>
```

Comment:
We don't seem to use title casing on most content headings (capital case on second word)